src/acl.c

```diff
@@ -1040,13 +1048,11 @@ int ACLSetUser(user *u, const char *op, ssize_t oplen) {
                 /* If user is trying to use the ACL mech to block SELECT except SELECT 0 or
                  * block DEBUG except DEBUG OBJECT (DEBUG subcommands are not considered
                  * subcommands for now) we use the allowed_firstargs mechanism. */
-                struct redisCommand *cmd = ACLLookupCommand(copy);
-                if (cmd == NULL) {
```

Comment:
can you explain why we needed that check in the past and not anymore?

Reply:
Going here means a base command with the first-arg (i.e. `select|1`), I think the reason is:
1. before this PR, `ACLLookupCommand(select|1)` return select command.
2. after this PR, `ACLLookupCommand(select|1)` return NULL.

Reply:
actually it was just a mistake i made recently in 43e736f79b
we already have the command struct of `copy`, no need to look it up again